packages/server/src/index.ts

```diff
@@ -133,6 +133,16 @@ io.on("connection", (socket) => {
       await socket.leave(topic);
     }
   });
+
+  socket.on("unique_subscribe", async ([game_id, seat, nr_of_players]) => {
```

Comment:
renamed

Reply:
Ah sorry for the confusion, I had deleted my comment around this, as I realized the original motivation better (`subscribe` is generic, `take_seat` is directly related to seats).  Either way is fine with me now, but feel free to push back if I ever comment something that doesn't make sense haha

Reply:
No problem, what do you think about the name "subscribe_to_unique_seat"? :-)

Reply:
Works for me!